<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MCMESHTRANGUI_ReadMCNPUmeshDlg</class>
 <widget class="QDialog" name="MCMESHTRANGUI_ReadMCNPUmeshDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>568</width>
    <height>260</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>568</width>
    <height>260</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Load File</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>531</width>
     <height>81</height>
    </rect>
   </property>
   <property name="title">
    <string>MCNP6 Umesh eeout file</string>
   </property>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>91</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Input File</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="rmu_le_umo">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>40</y>
      <width>281</width>
      <height>27</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>300</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="rmu_pb_umobrw">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>40</y>
      <width>96</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Browse</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="rmu_gb_rmml">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>551</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>Remove Multiplier</string>
   </property>
   <widget class="QRadioButton" name="rmu_rb_rmml_yes">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>112</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Yes</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="rmu_rb_rmml_no">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>30</y>
      <width>112</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>No</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="rmu_gb_kpInst">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>551</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>Keep Instances</string>
   </property>
   <widget class="QRadioButton" name="rmu_rb_kpInst_yes">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>112</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Yes</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="rmu_rb_kpInst_no">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>30</y>
      <width>112</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>No</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>220</y>
     <width>351</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>* The multiplier in total data set will not be removed</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
